packages/tag/src/index.vue

```diff
@@ -40,7 +40,7 @@ export default defineComponent({
   props: {
     closable: Boolean,
     type: {
-      type: String,
+      type: String as PropType<'success' | 'info' | 'warning' | 'danger'>,
```

Comment:
Probably adding empty string as one of the string constants is better but this is acceptable.

Reply:
😄 agree, i will add an empty string.